azext_edge/edge/params.py

```diff
@@ -1184,3 +1239,27 @@ def load_iotops_arguments(self, _):
             help="Fully qualified role definition Id in the following format: "
             "/subscriptions/{subscriptionId}/providers/Microsoft.Authorization/roleDefinitions/{roleId}",
         )
+
+    with self.argument_context("iot ops schema version") as context:
+        context.argument(
+            "version_name",
+            options_list=["--name", "-n"],
+            help="Schema version name.",
+            type=int
+        )
+        context.argument(
+            "schema_name",
+            options_list=["--schema"],
+            help="Schema name.",
+        )
+        context.argument(
+            "description",
+            options_list=["--desc"],
+            help="Description for the schema version.",
+        )
+        context.argument(
+            "schema_content",
+            options_list=["--content"],
+            help="File path containing or inline schema content.",
+            arg_group=None
```

Comment:
Was this mixed up with `--version-content` or did I hurt myself in my confusion?

Reply:
the content only is in the version. 

Schema create has it cause it makes the schema + version